crypto/crypto/src/traits.rs

```diff
@@ -91,10 +91,10 @@ impl<T: ValidKey> ValidKeyStringExt for T {}
 
 /// A type family for key material that should remain secret and has an
 /// associated type of the [`PublicKey`][PublicKey] family.
-pub trait PrivateKey: Sized {
+pub trait PrivateKeyExt: Sized {
```

Comment:
"Ext" usually stands for "extended" or "extension". What is extended here :)

Reply:
Ironically, I was doing this to avoid name collisions, but I actually don't really need to. 

I can revert, but we need to know if we're going this route : o